### PR TITLE
style: update heading font weights

### PR DESCRIPTION
**Pull request by ju-Skinner**
## Description
Headings will now use 100 based scale (100...900), where the body will now use *25 scale (125...925)

### Body - NO CHANGE
|  Before  |  After  |
|--------|--------|
|image|image|

### Headings
|  Before  |  After  |
|--------|--------|
|image|image|

## Testing in `sage-lib`
1. Navigate to Foundation -> Typography http://localhost:4000/pages/foundations/typography
2. Inspect Paragraph (*25) and Headings (*00), 

## Related
https://kajabi.atlassian.net/browse/DSS-720

### CODE REVIEW

**Review by pixelflips — APPROVED** [review]
🔥 Nice approach! LGTM!

**Review by QuintonJason — CHANGES_REQUESTED** [review]
Requesting changes because of the design decision to likely change fro SF Pro to Roboto. We don't know if these will remain valid

**Review by QuintonJason — APPROVED** [review]
Verified with design that the headings will remain *00s and body will remain *25s

File: packages/sage-assets/lib/stylesheets/tokens/_font_weight.scss [hunk 1]
```diff
@@ -15,14 +15,26 @@ $sage-font-weights: (
...
+
 ///
 /// Sage font weight token utility
 ///
 /// @param {string} $key [regular] The token to retrieve
 ///
 /// @return {number} The value retrieved
 ///
-@function sage-font-weight($key: regular) {
+@function sage-font-weight($key: regular, $heading: false) {
```
Comment (monicawheeler) [inline packages/sage-assets/lib/stylesheets/tokens/_font_weight.scss:32]: 🙌

**Review by monicawheeler — APPROVED** [review]

**Pull request merged**